### PR TITLE
Smpe vtl fix

### PR DESCRIPTION
**Pull request by abastyr**
Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Necessary documentation (if appropriate) have been added / updated
- [x] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [x] Bugfix 
- [ ] Feature 
- [ ] Other... Please describe:

#### Relevant issues

No related issue

#### Changes proposed in this PR

- Removed pointless `sysmodsCheck` variable
- Fixed some conditions for `ibmTemplate` to make sure conditions work properly even if the variable default is not initialized during the workflow execution
- Added extra conditional steps for SMP/E Apply and Accept that will be executed if checks work well.

#### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Does this PR do something the person installing Zowe should know about?

---
* Affected function: _general area of interest_ *

---
* Description: _1 line description_ *

---
* Part: _name of customizable file involved_  *

---
_multi-line description_

#### Is there a related doc issue or Pull Request? 

No

Doc issue/PR number:

### CODE REVIEW

File: smpe/ZOSMF/vtls/ZWE7APLY.vtl [hunk 1]
```diff
@@ -55,7 +55,7 @@
 //*
 //********************************************************************
 //*
-//APPLY    EXEC PGM=GIMSMP,REGION=0M
+//APPLY    EXEC PGM=GIMSMP,REGION=0M,COND=(4,LT)
```
Comment (OnnoVdT) [inline smpe/ZOSMF/vtls/ZWE7APLY.vtl]: as FYI, the point of the pointless condition is simplifying merging all SMPE jobs into a single one and halt on error.

**Comment by OnnoVdT** [comment]
You changed the FMID APPLY/ACCEPT vtls to do check and then conditionally do it again without check. However, the `#if($ibmTemplate == 'NO' || !$!ibmTemplate)` check is not present for the PTF APPLY/ACCEPT vtls.

File: smpe/ZOSMF/vtls/ZWE7APLY.vtl (hunk 1 above)
Comment (abastyr) [reply smpe/ZOSMF/vtls/ZWE7APLY.vtl]: I figured it was like that, however, the condition is "pointless" in the workflow :)

**Comment by abastyr** [comment]
I made the change for the workflow, so the APPLY/ACCEPT will always run with and without the check. We do not use the PTF vtls in the workflow :)

**Pull request merged**